packages/opencensus-exporter-stackdriver/src/common-utils.ts

```diff
@@ -29,7 +30,7 @@ export async function getDefaultResource(projectId: string):
   const [type, mappings] = getTypeAndMappings(autoDetectedResource.type);
   Object.keys(mappings).forEach((key) => {
     if (autoDetectedResource.labels[mappings[key]]) {
-      if (mappings[key] === resource.AWS_REGION_KEY) {
+      if (mappings[key] === CLOUD_RESOURCE.REGION_KEY) {
```

Comment:
You also need to check if the `CloudProvider` is AWS.

Reply:
Done, checking type is AWS_EC2_INSTANCE and key is REGION_KEY before adding `aws:` prefix. Sorry for the late response.